pkg/daemon/daemon.go

```diff
@@ -424,15 +426,22 @@ func (dn *Daemon) operatorConfigAddHandler(obj interface{}) {
 
 func (dn *Daemon) operatorConfigChangeHandler(old, new interface{}) {
 	newCfg := new.(*sriovnetworkv1.SriovOperatorConfig)
-	var level = glog.Level(newCfg.Spec.LogLevel)
-	if level != flag.Lookup("v").Value.(flag.Getter).Get() {
-		glog.Infof("Set log verbose level to: %d", level)
-		flag.Set("v", level.String())
-	}
+	dn.handleLogLevelChange(newCfg.Spec.LogLevel)
+
 	newDisableDrain := newCfg.Spec.DisableDrain
 	if dn.disableDrain != newDisableDrain {
 		dn.disableDrain = newDisableDrain
-		glog.Infof("Set Disable Drain to: %t", dn.disableDrain)
+		log.Log.Info("Set Disable Drain", "value", dn.disableDrain)
+	}
+}
+
+// handleLogLevelChange handles log level change
+func (dn *Daemon) handleLogLevelChange(logLevel int) {
+	newLevel := int8(logLevel * -1)
```

Comment:
perhaps you can use log.Log.V(logLevel).Enabled() to verify whether the new level is the same or not. You won't need to have a reference to the zap package.